pkg/visitors/count_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package visitors
+
+import (
+	"testing"
+)
+
+func TestCount(t *testing.T) {
+	f := parseImage(t)
```

Comment:
maybe we should move parseImage into another file rather than just keeping it in find_tests. maybe some file for visitor test helpers

Reply:
See #158